sg/exec.go

```diff
@@ -40,7 +41,7 @@ func (l *logWriter) Write(p []byte) (n int, err error) {
 	for in.Scan() {
 		line := in.Text()
 		if !l.hasFileReferences {
-			l.hasFileReferences = hasFileReferences(line)
+			l.hasFileReferences, line = hasFileReferences(line)
```

Comment:
I think this will mess with indentation now, since we do trimSpace in `hasFileReferences`